Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -264,22 +270,30 @@ public boolean delete(UUID uuid, Users users, JedisPool redisPool, Configuration
         }
 
         // SQL
-        if (sqlType == SQLType.MySQL) {
-            MySQL.delete(uuid, sql, storageConfigNode);
-        } else if (sqlType == SQLType.SQLite) {
-            SQLite.delete(uuid, sql, storageConfigNode);
+        if (ConfigUtil.getSQLType() == SQLType.MySQL) {
+            MySQL.delete(uuid, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode());
+        } else if (ConfigUtil.getSQLType() == SQLType.SQLite) {
+            SQLite.delete(uuid, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode());
         }
 
         // Redis
-        Redis.delete(uuid, redisPool, redisConfigNode);
+        Redis.delete(uuid, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode());
 
         // RabbitMQ
-        RabbitMQ.delete(uuid, rabbitConnection);
+        rabbitDelete(uuid);
 
         return true;
     }
 
-    public static void delete(UUID uuid, SQL sql, ConfigurationNode storageConfigNode, SQLType sqlType) {
+    private void rabbitDelete(UUID uuid) {
+        try {
+            RabbitMQ.delete(uuid, getRabbitConnection());
```

Comment:
Might as well make this a new `delete` method if we're going to have a `broadcast` method. Redis & Rabbit both.